src/app/modules/layout/layout.component.html

```diff
@@ -0,0 +1,27 @@
+<mat-sidenav-container>
+  <mat-sidenav class="layout" [(opened)]="opened">
+    <div class="sidenav-header">
+      <p class="text-uppercase">gmfctn!</p>
+
+      <button class="example-icon" mat-icon-button aria-label="Example icon-button with menu icon" (click)="opened=!opened">
```

Comment:
"Example icon-button with menu icon" - are you sure that is the correct label?

Reply:
fixed